astropy/io/fits/diff.py

```diff
@@ -253,8 +251,7 @@ def __init__(self, a, b, ignore_hdus=[], ignore_keywords=[],
             are considered to be different.
             The underlying function used for comparison is `numpy.allclose`.
 
-            .. versionchanged:: 2.0
```

Comment:
I'm not sure if we should change this `versionchanged` to `versionadded`.

Reply:
Not sure either, but as I wanted to remove all references to ``tolerance`` keeping ``versionchanged`` did not make sense, so changing to ``versionadded`` did seem a good solution. This is also consistent with the ``atol`` description.